lms/static/js/discussions_management/views/discussions.js

```diff
@@ -80,6 +87,50 @@
                     this.notification.render();
                 },
 
+                cohortStateUpdate: function(state) {
+                    if ($('.discussions-management').data('enrollment-track-count') <= 1) {
+                        this.showDiscussionManagement(state.is_cohorted);
+                    } if (this.getSelectedScheme() !== COHORT) {
+                        this.showCohortSchemeControl(state.is_cohorted);
+                    }
+                },
+
+                showDiscussionManagement: function(show) {
+                    if (!show) {
+                        $('.btn-link.discussions_management').addClass(HIDDEN_CLASS);
+                        $('#discussions_management').addClass(HIDDEN_CLASS);
+                    } else {
+                        $('.btn-link.discussions_management').removeClass(HIDDEN_CLASS);
+                        $('#discussions_management').removeClass(HIDDEN_CLASS);
+                    }
+                },
+
+                showCohortSchemeControl: function(show) {
+                    if (!show) {
+                        $('.division-scheme-item.cohort').addClass(HIDDEN_CLASS);
+                        this.updateSchemeSelectionLayout(2);
+                    } else {
+                        $('.division-scheme-item.cohort').removeClass(HIDDEN_CLASS);
+                        this.updateSchemeSelectionLayout(3);
+                    }
+                },
+
+                updateSchemeSelectionLayout: function(columns) {
+                    if (columns === 2) {
+                        $('.division-scheme-item.none').removeClass(THREE_COLUMN_CLASS).addClass(TWO_COLUMN_CLASS);
```

Comment:
I believe you can do all 3 of these with a single line, using just the selector $('.division-scheme-item'). JQuery can operate on multiple returned values.